cleverhans/attacks.py

```diff
@@ -25,16 +363,17 @@ def fgsm(x, predictions, eps, back='tf', clip_min=None, clip_max=None):
                     value for components of the example returned
     :return: a tensor for the adversarial example
     """
+    warnings.warn("attacks.fgsm is deprecated and will be removed on 09-27-17."
+                  " Instantiate an attack object from FastGradientMethod.")
     if back == 'tf':
```

Comment:
This would require changing the function interface (because FastGradientMethod needs a model instead of the output tensor)

Do you prefer leaving it like this for backward compatibility or moving to the new interface?